stravalib/model.py

```diff
@@ -1,7 +1,12 @@
 """
```

Comment:
Here i've added a more robust module header (feel free to edit!).

Reply:
Just added a commit with some edits. This is good. We now have a single source of truth.